ci/matrix.xml

```diff
@@ -183,6 +182,8 @@
         'set scalaJSStage in Global := FullOptStage' \
         ++$scala javalibExTestSuite/test &&
     sbtretry ++$scala testSuite/test:doc compiler/test reversi/fastOptJS reversi/fullOptJS &&
+    sbtretry ++$scala compiler/compile:doc library/compile:doc javalibEx/compile:doc \
+        testInterface/compile:doc jasmineTestFramework/compile:doc stubs/compile:doc &&
```

Comment:
I see... stubs ended up in here... I merely copied what you wrote.

Stubs belongs in the other task. The rationale is that stuff in the other task does not depend on the Scala.js compiler itself. Therefore we can compile them independently (and not compile the Scala.js compiler at all).